neuralmonkey/nn/projection.py

```diff
@@ -18,15 +19,15 @@ def linear(inputs, size, scope="LinearProjection"):
     Returns:
         A tensor of shape batch x size
     """
-    with tf.variable_scope(scope):
+    with tf.variable_scope(scope) as varscope:
         if isinstance(inputs, list):
             # if there is a list of tensor on the input, concatenate along
             # the last dimension and project.
             inputs = tf.concat(inputs, axis=-1)
 
         return tf.contrib.layers.fully_connected(
             inputs, size, biases_initializer=tf.zeros_initializer(),
-            activation_fn=None, scope=scope)
+            activation_fn=None, scope=varscope)
```

Comment:
nejsem si jistej, jestli tohle náhodou taky nerozseká zpětnou kompatibilitu modelů. Spíš bych se na to vykašlal dělat tuhle změnu.